studs_pattern:
  studs: [[0, 0, 0, 0, 0], [0, 1, 0, 1, 0],  [0, 0, 1, 0, 0], [0, 1, 0, 1, 0]]
  distance: 0.3
  proximity: 0.05
  press: 20.0
  laser_threshold: 180
